Fix error when splitting DateTime format string

labkey-danield opened the pull request:
#### Rationale
Fixing a bad string split.

#### Related Pull Requests
- none

#### Changes
- Be smarter about splitting a DateTime format string into a date part and a time part.
- Remove unused 'throw' annotation.

labkey-danield committed:
Fix error when splitting a DateTime format.
Remove unused 'throws' annotation.

labkey-chrisj commented on src/org/labkey/test/components/domain/DomainFormPanel.java:
```diff
@@ -139,21 +139,28 @@ else if (fieldDefinition.getType().equals(FieldDefinition.ColumnType.Time))
-                // a data format may include spaces (time does not).
-                String format = fieldDefinition.getFormat().trim();
-                int index = format.lastIndexOf(" ");
+                // Identify the part of the format that is the date and the part that is the time.
```
Ah, that explains it.  
This is probably out of scope for this fix, but this code (parsing date format/time format out of a format string) seems like it ought to be in a utils class like `TestDateUtils.java`

labkey-danield committed:
Fix error when splitting a DateTime format.
Remove unused 'throws' annotation.
Move the mouse after clicking a project menu item.
Wait for an element after using popLocation.
Adjust expected date format.
